Sources/Valet/SinglePromptSecureEnclaveValet.swift

```diff
@@ -18,6 +18,8 @@
 //  limitations under the License.
 //
 
+#if os(iOS) || os(macOS)
```

Comment:
In our manually managed project we'd previously excluded this file. However, `Package.swift` has no way to exclude a single file in `Sources/Valet` from only a single target. So now we include this file (in both `Package.swift` and `Valet.xcodeproj`), but compile out the contents.